crates/transaction-emitter-lib/src/emitter/account_minter.rs

```diff
@@ -95,14 +93,14 @@ impl<'t> AccountMinter<'t> {
         let coins_per_seed_account = (expected_children_per_seed_account as u64)
             .checked_mul(
                 coins_per_account
-                    + req.max_gas_per_txn * req.gas_price * req.init_gas_price_multiplier,
+                    + req.init_max_gas_per_txn * req.gas_price * req.init_gas_price_multiplier,
```

Comment:
init_max_gas_per_txn can now be different/larger, so we need to account for that differently here